spec.bs

```diff
@@ -945,17 +945,16 @@ A <dfn export>fenced frame config instance</dfn> is a struct with the following
     :: |config|'s [=fenced frame config/embedder shared storage context=]
 </div>
 
-Each [=navigable=] has a <dfn for=navigable>fenced frame config instance</dfn>, which is a [=fenced
-frame config instance=] or null, initially null.
+Each [=browsing context=] has a <dfn for="browsing context">fenced frame config instance</dfn>,
```

Comment:
What is the significance of [=browsing context=] here vs [=browsing context group=] above?

Reply:
OK, whatever